/*
 * This file was automatically generated by EvoSuite
 * Sun Aug 23 21:44:00 GMT 2020
 */

package com.br.rfs.desafio.itau;

import org.junit.Test;
import static org.junit.Assert.*;
import com.br.rfs.desafio.itau.Application;

public class Application_ESTest {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Application application0 = new Application();
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      String[] stringArray0 = new String[7];
      // Undeclared exception!
      try { 
        Application.main(stringArray0);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
      }
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      // Undeclared exception!
      try { 
        Application.main((String[]) null);
        fail("Expecting exception: IllegalArgumentException");
      
      } catch(IllegalArgumentException e) {
         //
         // Args must not be null
         //
      }
  }

 
}
